client/src/main/java/org/spine3/base/Stringifiers.java

```diff
@@ -161,4 +167,129 @@ protected CommandId doBackward(String s) {
+     * should be registered in the {@code StringifierRegistry} class
+     * for the correct usage of the {@code MapStringifier} converter.
+     *
+     * @param <K> the type of the keys in the map
+     * @param <V> the type of the values in the map
+     */
+    protected static class MapStringifier<K, V> extends Stringifier<Map<K, V>> {
+
+        private static final char DEFAULT_ELEMENTS_DELIMITER = ',';
+        private static final String ESCAPE_SEQUENCE = "\\";
+        private static final String KEY_VALUE_DELIMITER = ESCAPE_SEQUENCE + ':';
+
+        /**
+         * The delimiter for the passed elements in the {@code String} representation,
+         * {@code DEFAULT_ELEMENTS_DELIMITER} by default.
+         */
+        private final String delimiter;
+        private final Class<K> keyClass;
+        private final Class<V> valueClass;
+
+        public MapStringifier(Class<K> keyClass, Class<V> valueClass) {
+            super();
+            this.keyClass = keyClass;
+            this.valueClass = valueClass;
+            this.delimiter = ESCAPE_SEQUENCE + DEFAULT_ELEMENTS_DELIMITER;
+        }
+
+        /**
+         * That constructor should be used when need to use
+         * a custom delimiter of the elements during conversion.
+         *
+         * @param keyClass   the class of the key elements
+         * @param valueClass the class of the value elements
+         * @param delimiter  the delimiter for the passed elements via string
+         */
+        public MapStringifier(Class<K> keyClass, Class<V> valueClass, String delimiter) {
+            super();
+            this.keyClass = keyClass;
+            this.valueClass = valueClass;
+            this.delimiter = ESCAPE_SEQUENCE + delimiter;
+        }
+
+        @Override
+        protected String doForward(Map<K, V> map) {
+            final String result = map.toString();
+            return result;
+        }
+
+        @Override
+        protected Map<K, V> doBackward(String s) {
+            final String[] buckets = s.split(Pattern.quote(delimiter));
+            final Map<K, V> resultMap = newHashMap();
+
+            for (String bucket : buckets) {
+                saveConvertedBucket(resultMap, bucket);
+            }
+            Ints.stringConverter();
+            return resultMap;
+        }
+
+        private Map<K, V> saveConvertedBucket(Map<K, V> resultMap, String element) {
+            final String[] keyValue = element.split(Pattern.quote(KEY_VALUE_DELIMITER));
+            checkKeyValue(keyValue);
+
+            final String key = keyValue[0];
+            final String value = keyValue[1];
+
+            try {
+                final K convertedKey = convert(keyClass, key);
+                final V convertedValue = convert(valueClass, value);
+                resultMap.put(convertedKey, convertedValue);
+                return resultMap;
+            } catch (Throwable ignored) {
+                throw conversionArgumentException(CONVERSION_EXCEPTION);
```

Comment:
That branch is tests with the `throw_exception_when_passed_parameter_does_not_match_expected_format` test method.